tests/console/postfix_tls.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Test Postfix mail server with SSL enabled
+# Note: The test case can be run separately for postfix sanity test,
+#       or run as stand-alone mail server (together with dovecot)
+#       in multi-machine test scenario if MAIL_SERVER var set.
+# Maintainer: Qingming Su <qmsu@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use mailtest;
+
+sub run {
+    my $self         = shift;
+    my $postfix_conf = "/etc/postfix/main.cf";
+    my $postfix_cert = "/etc/postfix/ssl/postfix.crt";
+    my $postfix_key  = "/etc/postfix/ssl/postfix.key";
+
+    select_console "root-console";
+    prepare_mail_server;
+
+    # Install postfix and required packages
+    zypper_call "in postfix cyrus-sasl cyrus-sasl-saslauthd mailx";
+
+    # Configure postfix with TLS support (only smtpd)
+    assert_script_run "curl " . data_url('postfix/main.cf') . " -o $postfix_conf";
+    assert_script_run "curl " . data_url('openssl/mail-server-cert.pem') . " -o $postfix_cert";
+    assert_script_run "curl " . data_url('openssl/mail-server-key.pem') . " -o $postfix_key";
+    assert_script_run "sed -i 's/^#tlsmgr/tlsmgr/' /etc/postfix/master.cf";
+    assert_script_run "systemctl restart saslauthd.service";
+    assert_script_run "systemctl is-active saslauthd.service";
+    assert_script_run "systemctl restart postfix.service";
+    assert_script_run "systemctl is-active postfix.service";
+
+    # Print service status for debugging
+    script_run "systemctl -l --no-pager status saslauthd.service";
+    script_run "systemctl -l --no-pager status postfix.service";
```

Comment:
Same as above, use wrapper function.

Reply:
Thanks for the comments, I will change the case accordingly.